libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -253,14 +253,36 @@ private IdentityClaim getIdentityClaim(
 
     private Optional<IdentityClaim> generateIdentityClaim(List<VcStoreItem> successfulVCStoreItems)
             throws HttpResponseExceptionWithErrorBody {
+        if (successfulVCStoreItems.isEmpty()) {
+            LOGGER.warn("Failed to generate identity claim");
+            return Optional.empty();
+        }
+
+        List<IdentityClaim> identityClaims = new ArrayList<>();
         for (VcStoreItem item : successfulVCStoreItems) {
             if (EVIDENCE_CRI_TYPES.contains(item.getCredentialIssuer())) {
-                return Optional.of(
-                        getIdentityClaim(item.getCredential(), item.getCredentialIssuer(), false));
+                identityClaims.add(
+                        getIdentityClaim(item.getCredential(), item.getCredentialIssuer(), true));
```

Comment:
We're now passing `true` here to avoid an error in the case of a missing property, and just get an empty collection back instead. This means that all uses of `getIdentityClaim` are now passing in true. We should probably refactor it out to make the method and signature simpler.